kernels/R.svg

```diff
@@ -0,0 +1 @@
+<html lang="en">  <head> <meta content="origin" name="referrer">  <script nonce="8RGD/Pg9mJ5CByzbnCFKMw==">window.google = {};(function(){var e=function(){this.g="";this.h=c};e.prototype.i=!0;var f=/^(?:(?:https?|mailto|ftp):|[^:/?#]*(?:[/?#]|$))/i,c={},g=function(b){var d=new e;d.g=b;return d};g("about:blank");google.navigateTo=function(b,d,a){b!=d&&b.google?b.google.r&&(b.google.r=0,b=b.location,a instanceof e||a instanceof e||(a="object"==typeof a&&a.i?a.g:String(a),f.test(a)||(a="about:invalid#zClosurez"),a=g(a)),b.href=a instanceof e&&a.constructor===e&&a.h===c?a.g:"type_error:SafeUrl",d.location.replace("about:blank")):d.location.replace(a)};}).call(this);(function(){var redirectUrl='https://commons.wikimedia.org/wiki/File:R_logo.svg';google.navigateTo(parent,window,redirectUrl);})();</script> <noscript> <META content="0;URL=https://commons.wikimedia.org/wiki/File:R_logo.svg" http-equiv="refresh"></META> </noscript> </head>  </html>
```

Comment:
something is wrong with the svg. Can we use this logo instead: https://github.com/JuniperKernel/JuniperKernel/blob/master/extras/juniper_logo.png ?

Reply:
the current svg seems to be some kind of preview script from google.

Reply:
Yeah I was expeditive, there you go.